app/models/volunteer_op.rb

```diff
@@ -3,10 +3,18 @@ class VolunteerOp < ActiveRecord::Base
   validates :title, :description, presence: true
   validates :organisation_id, presence: true, if: "source == 'local'"
   belongs_to :organisation
-
+  
+  geocoded_by :full_address
+  after_validation :geocode, if: :different_address?   
+  after_validation :clear_lat_lng
```

Comment:
ah, that makes some sense, but not so transparent for user - I wonder if it would make it clear to the user if those fields were filled in with grey the address of the org?  on focus they would be deleted - using the placeholder tag - know what I mean?